cmd/boots/dhcp.go

```diff
@@ -86,7 +86,9 @@ func (d dhcpHandler) serveDHCP(w dhcp4.ReplyWriter, req *dhcp4.Packet) {
 		trace.WithAttributes(attribute.String("CircuitID", circuitID)),
 	)
 
-	j, err := job.CreateFromDHCP(ctx, mac, gi, circuitID)
+	// otel: if the hw record this retrieves has a traceparent, the job code will make a span
+	// link and swap out the traceparent in the context from here on
```

Comment:
This comment actually belongs in `job.CreateFrom*`'s doc string instead of (or in addition to) here.